File: flight1.cpp
#include <cpp_api/navigation_bridge.h> //API

Navigation nav;
int main(int argc, char *argv[]){
    nav.take_off(5.0); // drone take off 
    nav.position_set(6.5,0,0); //first position setpoint
    nav.position_set(0,6.5,0); //second position setpoint
    nav.position_set(-6.5,0,0); // third position setpoint
    nav.position_set(0,-6.5,0); //fourth position setpoint
    nav.land(false); //Land
}

File: flight1.py
#!/usr/bin/env python
import time
from flyt_python import api
drone = api.navigation(timeout=120000) 

# 4 second gap to initalize all the subsytems
time.sleep(4)

print 'taking off'
drone.take_off(5.0) 

drone.position_set(6.5,0,0,relative=True) # for first position
drone.position_set(0,6.5,0,relative=True) #for second position 
drone.position_set(-6.5,0,0,relative=True) #for third postion 
drone.position_set(0,-6.5,0,relative=True) #for fourth postition

print 'Landing'  
drone.land(async=False)

#shutdown the instance
drone.disconnect()
